eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/components/soap/SOAPBindingReferenceComposite.java

```diff
@@ -202,6 +203,11 @@ public void widgetSelected(final SelectionEvent e) {
         upChxGD.horizontalSpan = 3;
         _unwrappedPayloadCheckbox.setLayoutData(upChxGD);
 
+        _copyNamespacesCheckbox = createCheckbox(composite, "Copy Namespaces");
+        GridData cnChxGD = new GridData(GridData.FILL_HORIZONTAL);
```

Comment:
I prefer breaking it out into more readable lines myself, but I've seen the form you are suggesting. John uses it. Is it more efficient? Or is this merely a question of style?